xml/x509_subjects.xml

```diff
@@ -0,0 +1,950 @@
+    <param pos="0" name="hw.device" value="Printer"/>
+    <param pos="0" name="hw.vendor" value="HP"/>
+    <param pos="0" name="hw.family" value="JetDirect"/>
+    <param pos="0" name="os.device" value="Printer"/>
+    <param pos="0" name="os.vendor" value="HP"/>
+    <param pos="0" name="os.family" value="JetDirect"/>
+    <param pos="1" name="host.mac"/>
+    <param pos="2" name="hw.product"/>
+    <param pos="2" name="os.product"/>
+  </fingerprint>
+  <fingerprint pattern="^CN=([a-zA-Z0-9\.\-\_]+),OU=HP-IPG,O=HP,L=Vancouver,ST=Washington,C=US$">
+    <description>HP Jet Direct</description>
+    <example host.name="HPD49F21">CN=HPD49F21,OU=HP-IPG,O=HP,L=Vancouver,ST=Washington,C=US</example>
+    <param pos="0" name="hw.device" value="Printer"/>
+    <param pos="0" name="hw.vendor" value="HP"/>
+    <param pos="0" name="hw.family" value="JetDirect"/>
+    <param pos="0" name="os.device" value="Printer"/>
+    <param pos="0" name="os.vendor" value="HP"/>
+    <param pos="0" name="os.family" value="JetDirect"/>
+    <param pos="1" name="host.name"/>
+  </fingerprint>
+  <fingerprint pattern="^CN=(?:Sourcefire3D|firepower|ciscoasa),OU=Intrusion Management System,O=(?:Sourcefire\\, Inc.|Cisco Systems\\, Inc),C=US$">
+    <description>Cisco Firepower</description>
+    <example>CN=firepower,OU=Intrusion Management System,O=Cisco Systems\, Inc,C=US</example>
+    <example>CN=ciscoasa,OU=Intrusion Management System,O=Cisco Systems\, Inc,C=US</example>
+    <example>CN=Sourcefire3D,OU=Intrusion Management System,O=Sourcefire\, Inc.,C=US</example>
+    <param pos="0" name="hw.device" value="IPS"/>
+    <param pos="0" name="hw.vendor" value="Cisco"/>
+    <param pos="0" name="hw.product" value="Firepower"/>
+    <param pos="0" name="hw.cpe23" value="cpe:/h:cisco:firepower:-"/>
+    <param pos="0" name="os.device" value="IPS"/>
+    <param pos="0" name="os.vendor" value="Cisco"/>
+    <param pos="0" name="os.product" value="Firepower"/>
+    <param pos="0" name="os.family" value="Linux"/>
+  </fingerprint>
+  <fingerprint pattern="^CN=ASA Temporary Self Signed Certificate$">
+    <description>Cisco ASA</description>
+    <example>CN=ASA Temporary Self Signed Certificate</example>
+    <param pos="0" name="hw.device" value="Firewall"/>
+    <param pos="0" name="hw.vendor" value="Cisco"/>
+    <param pos="0" name="hw.product" value="Adaptive Security Appliance"/>
+    <param pos="0" name="hw.cpe23" value="cpe:/h:cisco:adaptive_security_appliance:-"/>
+  </fingerprint>
+  <fingerprint pattern="^CN=synology\.com.*,O=Synology Inc\.,L=Taipei.*,C=TW$">
+    <description>Synology NAS</description>
+    <example>CN=synology.com,OU=FTP Team,O=Synology Inc.,L=Taipei,ST=Taiwan,C=TW</example>
+    <example>CN=synology.com,O=Synology Inc.,L=Taipei,C=TW</example>
+    <param pos="0" name="hw.vendor" value="Synology"/>
+    <param pos="0" name="hw.family" value="DiskStation"/>
```

Comment:
I wonder if we can find an example of this from one of their routers to see if this pattern is `Diskstation` only.

Reply:
The security gateways use as TLS subject like `CN=UBNT Router UI,O=Ubiquiti Networks,L=San Jose,ST=CA,C=US` instead.